generator/mavgen_c.py

```diff
@@ -16,8 +20,8 @@ def generate_version_h(directory, xml):
     f = open(os.path.join(directory, "version.h"), mode='w')
     t.write(f,'''
 /** @file
- *	@brief MAVLink comm protocol built from ${basename}.xml
- *	@see http://mavlink.org
+ *  @brief MAVLink comm protocol built from ${basename}.xml
+ *  @see http://mavlink.org
  */
```

Comment:
nearly all the changes in mavgen_c.py aren't in the python code but instead are in the formatting of the generated C code. I'd prefer to leave the formatting of the C alone unless there is some good reason to change it (I tried to make the generated C code as readable as I could)